src/pages/Book/book.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import axios from "axios";
+import { Cover } from "../../components/Cover/cover";
+import { Title } from "../../components/Title/title";
+import { Author } from "../../components/Author/author";
+import { Description } from "../../components/Description/description";
+import { Container } from "./book.styles";
+import { CoverDiv } from "./book.styles";
+
+export class Book extends React.Component {
+  state = {
+    data: null,
+    isLoading: false,
+  };
+
+  getBook = async (isbn) => {
+    try {
+      this.setState({ isLoading: true });
+      const { data } = await axios(
+        `https://www.googleapis.com/books/v1/volumes?q=isbn:${isbn}&key=${process.env.REACT_APP_API_KEY}`
```

Comment:
process.env.REACT_APP_API_KEY, maybe move this in a variable it will be easier to read the url